compiler/pipes/calc-func-dep.cpp

```diff
@@ -49,7 +49,9 @@ VertexPtr CalcFuncDepPass::on_enter_vertex(VertexPtr vertex) {
   //NB: There is no user functions in default values of any kind.
   if (auto call = vertex.try_as<op_func_call>()) {
     FunctionPtr other_function = call->func_id;
-    data.dep.push_back(other_function);
+    if (!other_function->is_extern() || other_function->is_resumable) {
```

Comment:
Please, add a comment why we skip resumable functions.

Reply:
Added a large comment above